### ISSUE TITLE
Method Not Allowed when edit machine name in Machine Details screen

### ISSUE THREAD

**Issue by xkvnn** [tale/headplane#20]
How to reproduce
1. Select a machine to go to Machine Details
2. Click `Edit machine name`
3. Rename machine name and click `Rename`
4.  You will got `405 Method Not Allowed` error

In Console tab show this one
```
Route "routes/_data.machines.$id" does not have an action, but you are trying to submit to it. To fix this, please add an `action` function to the route
```

**Comment by tale** [comment]
Thank you for mentioning, I'll look into it.

**Comment by tale** [comment]
Fixed in dca5f9d1493a0a629a960621a2417f8a646961e3.
Available in `0.2.0`
